themes/red-theme/lms/templates/header.html

```diff
@@ -43,7 +43,7 @@
     <h1 class="logo">
       <a href="${marketing_link('ROOT')}">
         <%block name="navigation_logo">
-            <img src="${static.url("images/logo.png")}" alt="${platform_name()}"/>
+            <img src="${static.url("images/logo.png")}" alt="${_("{platform_name} Home Page").format(platform_name=settings.PLATFORM_NAME)}"/>
```

Comment:
@singingwolfboy You wrote this, and I'm curious why you did `platform_name()` instead of how the rest of our files do `settings.PLATFORM_NAME`.

@cptvitamin I'm going to advocate just using your change and it can be altered later as needed.